floodviz/views.py

```diff
@@ -1,7 +1,80 @@
 from flask import request, render_template, abort
 
+# below is for data processing #
+import json
+import time
+from datetime import datetime
+import requests
+import operator
+# # # # # # # # # # # # # # # # # #
+
 from . import app
 
+
 @app.route('/home/')
 def home():
-    return render_template('index.html')
\ No newline at end of file
+    # new data store #
+    newd = []
+    sites_value_maxs = {}
+    sites = app.config['SITE_IDS']
+    start_date = app.config['START_DT']
+    end_date = app.config['END_DT']
+    n_show_series = app.config['N_SERIES']
+    hydro_meta = app.config['HYDRO_META']
+
+    # Set up to retrieve all site ids from url? #
+    for idx, site in enumerate(sites):
+        url = 'https://nwis.waterservices.usgs.gov/nwis/iv/?site=' + site + '&startDT=' + \
```

Comment:
You should put this service endpoint in the main config.py. Probably only https://nwis.waterservices.usgs.gov/nwis as the rest could vary if we decide to pull down different kinds of data.